com.cubrid.common.ui/src/com/cubrid/common/ui/spi/util/CommonUITool.java

```diff
@@ -358,12 +359,19 @@ public static GridLayout createGridLayout(int columns) {
 	 */
 	public static int openMsgBox(Shell sh, int dialogImageType, String title, String msg,
 			String[] dialogButton) {
+		return openMsgBox(sh, dialogImageType, title, msg, "", dialogButton);
+	}
+	
+	public static int openMsgBox(Shell sh, int dialogImageType, String title, String informativeMessage, String actualMessage,
```

Comment:
I am not sure about the way we handle informative message. please verify that all references of openInformationBox have the same behavior.